### PR TITLE
azure-identity 1.1.0

### PR DESCRIPTION
**Pull request by chlowell**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

**Comment by lmazuel** [comment]
@chlowell was introduced by this PR:
https://github.com/Azure/azure-sdk-for-python/pull/8346

Process is that master is dev version until it isn't

File: sdk/identity/azure-identity/HISTORY.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-### 1.1.0b1 Unreleased
+### 1.1.0 Unreleased
```
Comment (schaabs) [inline sdk/identity/azure-identity/HISTORY.md]: You should remove "Unreleased" yeah?

Comment (chlowell) [reply sdk/identity/azure-identity/HISTORY.md]: Yes, just waiting until I know the release date 😉

File: sdk/identity/azure-identity/README.md [hunk 2]
```diff
@@ -98,9 +98,9 @@ for more information.
 During local development on Windows, [DefaultAzureCredential][default_cred_ref]
 can authenticate using a single sign-on shared with Microsoft applications, for
 example Visual Studio 2019. This may require additional configuration when
-multiple identities have signed in. In that case, set the environment variable
-`AZURE_USERNAME` with the desired identity's username (typically an email
-address).
+multiple identities have signed in. In that case, set the environment variables
+`AZURE_USERNAME` and `AZURE_TENANT_ID` to select the desired identity. Either,
```
Comment (KieranBrantnerMagee) [inline sdk/identity/azure-identity/README.md]: I'm surprised we're removing the (typically an email address) clarity, I remember that crossing my mind when I first set up this type of credential.

**Pull request merged**